redact/main.py

```diff
@@ -97,6 +110,16 @@ def redact_file(
         auto_delete_job=auto_delete_job,
     )
 
+    if job_status is not None and job_status.state == JobState.failed:
+        log.info(f"Job failed for '{file_path}': {job_status.error}")
```

Comment:
Shouldn't we do at least warn level here?

Reply:
done